respy/tests/test_conditional_draws.py

```diff
@@ -1,136 +1,77 @@
+import pickle
+
 import numpy as np
 import pytest
-from numba import jit
-from scipy.stats import norm
-
-from respy.conditional_draws import kalman_update
+from numpy.testing import assert_array_almost_equal as aaae
+
+from respy.conditional_draws import calculate_conditional_draws2
+from respy.conditional_draws import update_cholcov
+from respy.conditional_draws import update_cholcov_with_measurement_error
+from respy.conditional_draws import update_mean_and_evaluate_likelihood
+from respy.config import HUGE_FLOAT
+from respy.config import TEST_RESOURCES_DIR
+
+
+def fixture():
+    """The inputs and outputs were generated using a well tested Kalman filter."""
+    with open(TEST_RESOURCES_DIR / "conditional_draws_fixture.pickle", "rb") as p:
+        fix = pickle.load(p)
+    return fix
+
+
+@pytest.mark.parametrize("i", range(20))
+def test_update_and_evaluate_likelihood(i):
+    fix = fixture()
```

Comment:
Having defined the fixture above, you can add the name of the fixture as an argument of the test function and access what the fixture returns. Here is the section in the docs: https://docs.pytest.org/en/latest/fixture.html.

Reply:
Done.